docs/osquery/osquery.asciidoc

```diff
@@ -51,14 +51,15 @@ and you'll get suggestions for agents by name, ID, platform, and policy.
 [role="screenshot"]
 image::images/enter-query.png[Select saved query dropdown name showing query name and description]
 
+. (Optional) Expand the **Advanced** section to view or set <<osquery-map-fields,mapped ECS fields>>. The results from the live query include the mapped fields.
+
 . Click **Submit**.
 
 . Review the results in a table, or navigate to *Discover* to dive deeper into the response,
 or to the drag-and-drop *Lens* editor to create visualizations.
 . To view more information about the request, such as failures, open the *Status* tab.
 . To optionally save the query for future use, click *Save for later* and define the ID,
-description, and other
-<<osquery-manage-query,details>>.
+description, and other <<osquery-manage-query,details>>. 
 
 To view a history of the past queries you have run, open the *Live queries history*.
```

Comment:
I ended up making it an independent sub-section, because I think it's a bit of a separate process from running a live query. Take a look and let me know whether you think the latest update works.